src/app/contextBroker/contextBroker.cpp

```diff
@@ -249,6 +250,7 @@ bool            mutexTimeStat;
 #define MAX_L               900000
 #define MUTEX_POLICY_DESC   "mutex policy (none/read/write/all)"
 #define MUTEX_TIMESTAT_DESC "measure total semaphore waiting time"
+#define WRITE_CONCERN_DESC  "db write concern (0:none,1:normal)"
```

Comment:
OK, true.
I took 'none' and 'normal' from the names of the defines in the C++ driver.